Antidote/DialingCallViewController.m

```diff
@@ -33,20 +29,9 @@ @interface DialingCallViewController ()
 
 @implementation DialingCallViewController
 
-#pragma mark - Life cycle
-
-- (instancetype)initWithChat:(OCTChat *)chat submanagerCalls:(OCTSubmanagerCalls *)manager
-{
-    OCTCall *call = [manager callToChat:chat enableAudio:YES enableVideo:NO error:nil];
-
-    if (! call) {
-        return nil;
-    }
+@dynamic delegate;
```

Comment:
If I remove this, it will give me a warning "Auto property synthesis will not synthesize property 'delegate'; it will be implemented by its superclass, use @dynamic to acknowledge intention" Which we do want, since we are assigning the delegate at the superclass `AbstractCallViewController`